src/org/labkey/test/components/ui/EntityBulkUpdateDialog.java

```diff
@@ -0,0 +1,205 @@
+package org.labkey.test.components.ui;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.bootstrap.ModalDialog;
+import org.labkey.test.components.glassLibrary.components.FilteringReactSelect;
+import org.labkey.test.components.html.Checkbox;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.components.html.ToggleButton;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+
+public class EntityBulkUpdateDialog extends ModalDialog
+{
+    public EntityBulkUpdateDialog(WebDriver driver)
+    {
+        this(new ModalDialogFinder(driver).withTitle("samples selected from"));
+    }
+
+    private EntityBulkUpdateDialog(ModalDialogFinder finder)
+    {
+        super(finder);
+    }
+
+    // enable/disable field editable state
+
+    public boolean isFieldEnabled(String columnTitle)
+    {
+        return elementCache().getToggle(columnTitle).get();
+    }
+
+    public EntityBulkUpdateDialog setEditableState(String columnTitle, boolean enable)
+    {
+        elementCache().getToggle(columnTitle).set(enable);
+        return this;
+    }
+
+    // interact with selection fields
+
+    public EntityBulkUpdateDialog setSelectionField(String columnTitle, List<String> selectValues)
+    {
+        setEditableState(columnTitle, true);
+        FilteringReactSelect reactSelect = elementCache().getSelect(columnTitle);
+        getWrapper().waitFor(()-> !reactSelect.isDisabled(),
+                "the ["+columnTitle+"] reactSelect did not become enabled in time", 2000);
+        selectValues.forEach(s -> {reactSelect.filterSelect(s);});
+        return this;
+    }
+
+    public List<String> getSelectionFieldValues(String fieldKey)
+    {
+        return elementCache().getSelect(fieldKey).getSelections();
+    }
+
+    public EntityBulkUpdateDialog setAliases(List<String> aliases)
```

Comment:
Why is there a specific field for Aliases when it is a selection field? Same can be asked about description. This should be a more generalized control and not assume that there are any pre-defined fields.

Reply:
I've assumed that these come standard with whatever set of fields you specify, since that's what I've been doing- creating a set of fields and getting them + these two.
I could pretty easily burn these down and make the tests do that work